lexicons/com/atproto/identity/defs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.defs",
+  "defs": {
+    "atprotoIdentity": {
+      "type": "object",
+      "required": ["did"],
+      "properties": {
+        "did": { "type": "string", "format": "did" },
+        "handle": {
+          "type": "string",
+          "format": "handle",
+          "description": "The validated handle of the account; or 'handle.invalid' if the handle did not bi-directionally match the DID document."
```

Comment:
is it out of character for this to be a magic string? instead leave the whole `handle` clause optional and only present if valid?

Reply:
or make an explicit `{"handle.invalid":true}` member to the result object?

Reply:
we do the `handle.invalid` thing pretty consistently across the stack: it is mentioned in the spec, processed specially in the app, etc.

arguably we might want to signal "this DID does not declare a handle at all" separately from "the handle doesn't validate bi-directionally", for things like service DIDs. but this is the established pattern at this point.